data/transactions/logic/eval.go

```diff
@@ -911,39 +938,18 @@ func opLt(cx *evalContext) {
 }
 
 func opGt(cx *evalContext) {
-	last := len(cx.stack) - 1
-	prev := last - 1
-	cond := cx.stack[prev].Uint > cx.stack[last].Uint
-	if cond {
-		cx.stack[prev].Uint = 1
-	} else {
-		cx.stack[prev].Uint = 0
-	}
-	cx.stack = cx.stack[:last]
+	opSwap(cx)
+	opLt(cx)
```

Comment:
I thought entire point of duplicating code in each opcode was in reducing number of additional operations.
Like `opLe` now is `opSwap + opLt + opNot` that is (5 + 6 + 3) = 14 instead of 6 in old version (counted mem read/write, swap is expensive since no direct mem to mem copy in a single cpu operation).
IMO this is questionable, maybe benchmark it?

Reply:
I did this when I found writing the equivalent byteslice operators to be tedious, repetitive, and error-prone.   It's hard for me to imagine any appreciable performance effect here, with all this data in cache.  I'll write a little benchmark to convince myself.

Reply:
Here are the result.  I put them in order of overhead introduced by the shorter, clearer code. Each one is on more function call than the last.

1. Benchmark <          16.9-17.8 ns/op
2. Benchmark >=         17.3-17.5 ns/op
3. Benchmark >          20.0-23.3 ns/op
4. Benchmark <=         21.8-22.0 ns/op


3 ns seems like an ok price to pay, for the largest overhead. 

The biggest jump is between 2 & 3, because it involves opSwap.  I suppose I could be convinced to repeat the code to eliminate that one.  Then they'd all be so close the difference is in the noise.

Reply:
Thank you for benchmarking it! I think I agree, 3ns/op is OK price for cleaner code.